### PR TITLE
feat(cloudwatch-put-metrics): Key metrics and monitoring on core func…

### PR DESCRIPTION
**Pull request by tx0c**
…tionalities

resolves thematters/developer-resource#303

### CODE REVIEW

**Review by gary02 — APPROVED** [review]

File: src/mutations/user/walletLogin.ts [hunk 1]
```diff
@@ -232,6 +233,18 @@ const _walletLogin: Exclude<
...
+    atomService.aws.putMetricData({
+      MetricData: [
+        {
+          MetricName: METRICS_NAMES.UserRegistrationCount,
+          Timestamp: new Date(),
+          Unit: 'Count',
+          Value: 1,
+        },
+      ],
+    })
```
Comment (robertu7) [inline src/mutations/user/walletLogin.ts:247]: and socialLogin?

**Review by robertu7 — APPROVED** [review]
and socialLogin?

File: src/mutations/user/walletLogin.ts (hunk 1 above)
Comment (tx0c) [reply src/mutations/user/walletLogin.ts]: revised to handle metrics emission in `userService` better

**Comment by tx0c** [comment]
> and socialLogin?

covered all entries in `userService` handling all user creation

**Pull request merged**